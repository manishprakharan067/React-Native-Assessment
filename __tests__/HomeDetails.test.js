import React from 'react';
import renderer from 'react-test-renderer';
import HomeDetailsScreen from '../screens/HomeDetails';

test('renders correctly', () => {
  const tree = renderer.create(<HomeDetailsScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
