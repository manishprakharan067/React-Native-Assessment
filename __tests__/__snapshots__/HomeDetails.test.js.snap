// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <RCTSafeAreaView
    emulateUnlessSupported={true}
  />,
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#877825",
        "borderRadius": 5,
        "height": 30,
        "justifyContent": "center",
        "marginLeft": 10,
        "marginTop": 10,
        "opacity": 1,
        "width": 50,
      }
    }
  >
    <Text>
      Back
    </Text>
  </View>,
  <View
    style={
      Array [
        Object {
          "flex": 1,
          "overflow": "hidden",
        },
        undefined,
      ]
    }
  >
    <RNCWebView
      cacheEnabled={true}
      injectedJavaScriptBeforeContentLoadedForMainFrameOnly={true}
      injectedJavaScriptForMainFrameOnly={true}
      javaScriptEnabled={true}
      messagingEnabled={true}
      onContentProcessDidTerminate={[Function]}
      onHttpError={[Function]}
      onLoadingError={[Function]}
      onLoadingFinish={[Function]}
      onLoadingProgress={[Function]}
      onLoadingStart={[Function]}
      onMessage={[Function]}
      onShouldStartLoadWithRequest={[Function]}
      source={
        Object {
          "uri": undefined,
        }
      }
      style={
        Array [
          Object {
            "flex": 1,
            "overflow": "hidden",
          },
          Object {
            "backgroundColor": "#ffffff",
          },
          undefined,
        ]
      }
      textInteractionEnabled={true}
      useSharedProcessPool={true}
    />
  </View>,
]
`;
